openpmd_viewer/openpmd_timeseries/data_reader/io_reader/utilities.py

```diff
@@ -146,10 +146,12 @@ def get_data(series, record_component, i_slice=None, pos_slice=None,
     if (output_type is not None) and (data.dtype != output_type):
```

Comment:
Is it possible we should do this conversion after the scaling?

Current version:
- flexible, will not truncate, but ignores `output_type` for `unit_SI` != 1.0

Convert later:
- honor `output_type` but potentially crop precision